src/app/containers/IndexPageSection/index.jsx

```diff
@@ -159,11 +159,16 @@ const sectionBody = ({
   return renderPromos(items, isFirstSection, dir);
 };
 
-const IndexPageSection = ({ bar, group, sectionNumber }) => {
+const IndexPageSection = ({
+  bar,
+  group,
+  sectionNumber,
+  renderWithoutStrapline,
+}) => {
   const { script, service, dir, translations } = useContext(ServiceContext);
   const sectionLabelId = idSanitiser(group.title);
 
-  const strapline = pathOr(null, ['strapline', 'name'], group);
+  const strapline = pathOr('', ['strapline', 'name'], group);
```

Comment:
Is there a specific reason why we're rendering an empty string here instead of `null` now?

Reply:
I believe it was causing issues with prop types on the Chartbeat component (which resulted in browser console errors in local dev environment). This was an easy way to resolve that.

Reply:
I think we might need to confirm what the values for those would be in the FIX pages data and set them appropriately in [this file](https://github.com/bbc/simorgh/blob/latest/src/app/containers/ChartbeatAnalytics/utils/index.js#L104-L125) for `getType` and `getTitle` appropriately

Reply:
Yes. We haven't quite got to that part yet - we were going to try and get it onto Test ASAP before we fix these issues.

Reply:
Checked out locally (25/09/20 11:34), I see the following warnings:
```
Warning: Failed prop type: The prop `config.title` is marked as required in `Chartbeat`, but its value is `null`.
Warning: Failed prop type: The prop `chartbeatConfig.title` is marked as required in `CanonicalChartbeatBeacon`, but its value is `null`.
```

Reply:
This issue will be fixed later as part of https://github.com/bbc/simorgh/issues/7910

Reply:
Once we solve the Chartbeat issue, we can revert the strapline to be `const strapline = pathOr(null, ['strapline', 'name'], group);`